package.json

```diff
@@ -12,10 +12,11 @@
     "docusaurus": "docusaurus",
     "lint": "eslint .",
     "lint-prose": "cp .proselintrc ~/ && proselint docs blog",
+    "prepare": "husky install",
     "start": "yarn styles && docusaurus start",
     "styles": "sass src/css/custom-bootstrap.scss src/css/bootstrap.css",
     "swizzle": "docusaurus swizzle",
-    "test": "yarn lint && yarn lint-prose && yarn build"
+    "test": "yarn lint && yarn build"
```

Comment:
The `"lint-prose"` script clobbers the home directory so it's not safe to include in the default `yarn test` command, especially since that gets called automatically by `git commit` via the Husky pre-commit hook.